enrolments/tests/test_serializers.py

```diff
@@ -217,14 +239,37 @@ def test_enrolment_serializer(self):
                     "year": self.session.year,
                 },
                 "preferred_class": {
-                    "id": self.preferred_class.id,
-                    "name": self.preferred_class.name,
+                    "id": self.class1.id,
+                    "name": self.class1.name,
                 },
                 "enrolled_class": {
-                    "id": self.enrolled_class.id,
-                    "name": self.enrolled_class.name,
+                    "id": self.class2.id,
+                    "name": self.class2.name,
                 },
                 "status": self.enrolment.status,
             },
             EnrolmentSerializer(self.enrolment).data,
         )
+
+    def test_enrolment_update(self):
+        serializer = EnrolmentSerializer(data=self.update_request)
+        self.assertTrue(serializer.is_valid())
+        updated_enrolment = EnrolmentSerializer.update(
+            self, self.enrolment, self.update_request
+        )
```

Comment:
it'd be nice to use `.save()` instead, with the actual serializer defined on line 255. it'll better represent the code that gets called by the endpoint (has a bunch more safety checks up front) - see deets [here](https://stackoverflow.com/a/45101051)
```suggestion
        serializer = EnrolmentSerializer(self.enrolment, data=self.update_request)
        self.assertTrue(serializer.is_valid())
        serializer.save()
```